content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,129 @@
+- Deleting users.
+
+Administrators with the appropriate permissions can perform these actions.
+
+== Accessing the Manage Users Page
+
+To manage users in Jenkins, you need to have administrative privileges. Follow these steps to access the user management section:
+
+. Log in to your Jenkins instance with an account that has administrative permissions.
++
+. Navigate to the **Manage Jenkins** page from the Jenkins dashboard.
++
+image::managing/manager-users-home-page.png[Accessing Manage Jenkins page,width=600]
+
+. Click on **Users** under the **Security** section.
++
+image::managing/select-users.png[Accessing the Users section,width=600]
+
+== Creating a User
+
+To create a new user in Jenkins:
+
+. On the **Users** page, click on the **Create User** button.
++
+image::managing/create-users-click.png[Accessing the Create User button,width=600]
+
+. Fill in the required details:
+   - **Username**: A unique identifier for the user.
+   - **Password**: A secure password for the user.
+   - **Confirm Password**: Re-enter the password.
+   - **Full Name**: The full name of the user.
+   - **Email Address**: The email address of the user.
++
+image::managing/create-users.png[Accessing the Create User Page,width=600]
+
+. Click **Create User** to save the new user.
++
+For example:
++
+image::managing/create-users-example.png[Create User Page Example,width=600]
+
+== Configuring User Settings
+
+Once a user is created, you can configure their settings:
+
+. On the **Manage Users** page, click on the user you want to configure.
++
+. Update the following details as needed:
+   - **Full Name**: Edit the user's full name.
+   - **Description**: Update the user's description.
+   - **Credentials**: Change the user's credentials.
++
+image::managing/account-preference.png[Configure page,width=600]
+
+. Click **Save** to apply the changes.
++
+image::managing/select-user.png[Configuring User page,width=600]
+
+== Modifying User Information
+
+Administrators can modify user information at any time:
+
+. Navigate to the **Manage Users** page.
++
+. Click on the user whose information you want to modify.
++
+. Update the relevant fields (e.g., full name, email address, password).
++
+. Click **Save** to apply the changes.
++
+
+== Deleting a User
+
+To remove a user from Jenkins:
+
+. Navigate to **Manage Users**.
++
+. Find the user you want to delete.
++
+. Click on the **Delete** button next to the user’s name.
++
+image::managing/delete-users.png[Deleting User,width=600]
+
+. Confirm the deletion.
++
+
+⚠️ **Note:** Deleting a user **permanently removes their account** from Jenkins. Make sure they don't have active jobs or responsibilities before proceeding.
+
+== Linking to Security Settings
+
+User management is closely tied to Jenkins' security settings. For more information on configuring permissions and authorization, refer to the link:../managing/security.adoc#authorization[Managing Security - Authorization] documentation.
```

Comment:
```suggestion
User management is closely tied to Jenkins' security settings. For more information on configuring permissions and authorization, refer to the link:/doc/book/security/access-control/permissions/[Access Control - Permissions] documentation.
```
The link currently configured does not lead to an actual page, this should leader to the Permissions page of access control, which would fit with the described content.